test/tests/types.test.js

```diff
@@ -1176,25 +1176,26 @@ describe('graphql - types parsing and validation', () => {
     describe('input literal', () => {
       test('cds.Timestamp is correctly parsed from input literal timestamp string value', async () => {
         const value = '2021-06-27T14:52:23.123Z'
+        const returnValue = /2021-06-27T14:52:23\.123(0000)?Z/ // timestamp precision increase with cds^7
         const query = _getMutationForFieldWithLiteralValue(field, value, true)
-        const data = { TypesService: { MyEntity: { create: [{ [field]: value }] } } }
+        const data = { TypesService: { MyEntity: { create: [{ [field]: expect.stringMatching(returnValue) }] } } }
         const response = await POST('/graphql', { query })
         expect(response.data).toEqual({ data })
 
         const result = await SELECT.from('sap.cds.graphql.types.MyEntity').columns(field)
-        expect(result).toContainEqual({ [field]: value })
+        expect(result).toContainEqual({ [field]: expect.stringMatching(returnValue) })
       })
 
-      test('cds.Timestamp is correctly parsed from input literal timestamp string value', async () => {
-        const value = '2021-06-27T14:52:23.123Z'
-        const returnValue = '2021-06-27T14:52:23.123Z'
+      test('cds.Timestamp is correctly parsed from input literal high precision timestamp string value', async () => {
+        const value = '2021-06-27T14:52:23.1234567Z'
```

Comment:
Is that really required? It should still be possible to insert with less precise values